src/modules/Layout/Header/Header.js

```diff
@@ -18,6 +24,12 @@ const Header = ({ logged }) => {
                     <Nav />
                 </div>
                 <div className={b('col', { right: true })}>
+                    {
+                        user &&
+                            <span className={b('user')}>
```

Comment:
maybe `user-name` or `display-name` would be more specific for this user sounds like a container imo.